plugins/woocommerce/includes/admin/wc-meta-box-functions.php

```diff
@@ -66,7 +67,13 @@ function woocommerce_wp_text_input( $field, WC_Data $data = null ) {
 	}
 
 	echo '<p class="form-field ' . esc_attr( $field['id'] ) . '_field ' . esc_attr( $field['wrapper_class'] ) . '">
-		<label for="' . esc_attr( $field['id'] ) . '">' . wp_kses_post( $field['label'] ) . '</label>';
+		<label ';
```

Comment:
Totally nitpicky. This looks good, but I wonder whether using a variable would make the code more legible.

It would look like this:
```
	$class = ! empty( $field['label_class'] ) ? 'class="' . esc_attr( $field['label_class'] ) . '" ' : '';

	echo '<p class="form-field ' . esc_attr( $field['id'] ) . '_field ' . esc_attr( $field['wrapper_class'] ) . '">
		<label ' . $class . ' for="' . esc_attr( $field['id'] ) . '">' . wp_kses_post( $field['label'] ) . '</label>';

```

But I'm totally fine leaving the code as it is now.